Makefile

```diff
@@ -1,27 +1,32 @@
 SENTRY_PATH := `python -c 'import sentry; print sentry.__file__.rsplit("/", 3)[0]'`
 UNAME_S := $(shell uname -s)
 
-develop: setup-git install-yarn
-	pip install "pip>=9,<10"
-	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional]
-	pip install -e ".[tests]"
+PIP_VERSION := 19.1.1
+PIP_OPTS := --no-use-pep517 --disable-pip-version-check
+
+develop: ensure-pip setup-git install-yarn
+	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional] $(PIP_OPTS)
+	pip install -e ".[tests]" $(PIP_OPTS)
 	yarn install
 
+ensure-pip:
+	pip install "pip==$(PIP_VERSION)"
```

Comment:
Ok, I read your suggested changes bottom-to-top. We just have a mixture of `python -m pip` and `pip`, and I agree that the former is better. Not so much for the windows reason, but because it ties the pip invocation to the correctness of what python is resolved to (pip could be anything.)

Going forward, since we both seem to have strong opinions, we (at least us) should collaborate on an initial internal draft of best practices like this, then gradually refactor everything over. Not saying we should go to the level of linting for this kind of stuff, but at least get all these opinions in 1 place.

Reply:
>Going forward, since we both seem to have strong opinions, we (at least us) should collaborate on an initial internal draft of best practices like this, then gradually refactor everything over. Not saying we should go to the level of linting for this kind of stuff, but at least get all these opinions in 1 place.

Agree with the second part but I don't have a strong opinion on any of these (would have requested changes if I did) :D

These are mere highlights or recommendations but as long as your version works and it is not "terrible", go ahead and merge :)

Reply:
Well, the opportunity to escape the consistency trap you mentioned elsewhere is already here, so I'll just apply your suggestions to this pr.